subprojects/docs/src/docs/userguide/javaTesting.adoc

```diff
@@ -398,3 +443,42 @@ Creating and configuring a source set automatically sets up the compilation stag
 ++++
 
 Again, we're accessing a source set to get the relevant information, i.e. where the compiled test classes are — the `testClassesDir` property — and what needs to be on the classpath when running them — `classpath`.
+
+Users commonly want to run integration tests after the unit tests, because they are often slower to run and you want the build to fail early on the unit tests rather than later on the integration tests. That's why the above example adds a `shouldRunAfter()` declaration. This is preferred over `mustRunAfter()` so that Gradle has more flexibility in executing the build in parallel.
+
+[[sec:skipping_java_tests]]
+=== Skipping the tests
+
+If you want to skip the tests when running a build, you have a few options. You can either do it via <<sec:excluding_tasks_from_the_command_line,command line arguments>> or <<sec:skipping_tasks,in the build script>>. To do it on the command line, you can use the `-x` or `--exclude-task` option like so:
+
+    gradle build -x test
+
+This excludes the `test` task and any other task that it _exclusively_ depends on, i.e. no other task depends on the same task. Those tasks will not be marked "SKIPPED" by Gradle, but will simply not appear in the list of tasks executed.
+
+Skipping a test via the build script can be done a few ways. One common approach is to make test execution conditional via the `onlyIf()` method. The following sample skips the `test` task if the project has a property called `mySkipTests`:
```

Comment:
❓ Is there a way we can link to some kind of API doc for `onlyIf`?

Reply:
Makes sense, and done.